README.md

```diff
@@ -276,6 +276,28 @@ client
   .catch(error => console.log(error.errorMessages))
 ```
 
+##### Create a Signed Search Key
+
+Creating a search key that will only return the title field.
+
+```javascript
+const readOnlyApiKey = 'search-xxxxxxxxxxxxxxxxxxxxxxxx'
```

Comment:
That is minor, but I would use the same words that are used in the app:
publicSearchKey or publicSearchApiKey
![image](https://user-images.githubusercontent.com/11838280/63541710-0d426f00-c4f5-11e9-8698-4c5f211bdfe7.png)
The current naming may lead to confusion because we have read/write mode for a private key:
<img width="963" alt="Screen Shot 2019-08-22 at 3 54 21 PM" src="https://user-images.githubusercontent.com/11838280/63541762-2cd99780-c4f5-11e9-88ca-5effb1d3de26.png">